### DISCUSSION TITLE
Unable to get Miryoku KMK to work with crowboard...

### DISCUSSION THREAD

**Discussion by hansamann** [manna-harbour/miryoku#195] [Q&A] [answered]
I tried KMK and after copying the main.py generated by the github workflow to a crowboard KMK installation nothing works... what could be the issue?

1) flashed micropython and copied KMK for crowboard from here: https://github.com/doesntfazer/Keyboard-Dweebs-Firmware-repository/tree/main/KMK/CrowBoard
NOTE: the micropython flashed here is not the latest but it works with the default setup crowboard has

2) keyboard works in this original config

3) forked miryoku kmk, activated workflows and ran build inputs... worked

4) downloaded artifact zip, copied main.py over to pi pico

5) does not work... :-(

**Comment by manna-harbour** [comment, marked as answer]
In general, see the last paragraph in this section: https://github.com/manna-harbour/miryoku_kmk#installation.

However, the crowboard `kb.py` looks like it would be compatible without modification.

What is the boot output? Are there any errors on the serial console? How about the MCU status lights? Also try enabling debugging.

You could also try editing the default `main.py` and replacing the keymap array and module imports with those from the generated file.

**Reply by hansamann** [reply]
It seems to work and I think the issue is a kmk lib issue. 
1) flashed latest circuitpython https://circuitpython.org/board/raspberry_pi_pico/
2) cloned latest kmk and copied kmk folder
3) copied kb.py from https://github.com/doesntfazer/Keyboard-Dweebs-Firmware-repository/tree/main/KMK/CrowBoard/Copy%20these%20files%20after%20Circuit%20python
4) copied new main.py

The difference is the kmk folder and the latest circuitpython... 

thx a lot!
